photon-client/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -14,28 +14,212 @@
+      "feature-name-and-term-set-path" -> "")  // required by GAMEParams parser, but not used in GAMEEstimator
+    val params = GameParams.parseFromCommandLine(CommonTestUtils.argArray(args))
+
+    // Compute normalization contexts based on statistics of the training data for this (unique) feature shard
+    val normalizationContexts: Option[Map[String, NormalizationContext]] =
+      Some(Map((featureShardId, stats))
+        .mapValues { featureShardStats =>
+          val intercept: Option[Int] = featureIndexMap.get(GLMSuite.INTERCEPT_NAME_TERM)
+          NormalizationContext(params.normalizationType, featureShardStats, intercept)
+        })
+
+    // Create GAMEEstimator and fit model
+    // Returns (model, evaluation, optimizer config)
+    val models: Seq[(GAMEModel, Option[EvaluationResults], String)] =
+        createEstimator(params, "simpleTest")._1.fit(trainingData, validationData = None, normalizationContexts)
+
+    val model = models.head._1.getModel(coordinateId).head.asInstanceOf[FixedEffectModel].model
+
+    // Reference values from scikit-learn
+    assertEquals(model.coefficients.means(0), 0.3215554473500486, 1e-12)
+    assertEquals(model.coefficients.means(1), 0.17904355431985355, 1e-12)
+    assertEquals(model.coefficients.means(2), 0.4122241763914806, 1e-12)
+  }
+
+  /**
+   * Build GAMEParams for each possible value of the normalization.
+   *
+   * @return An collection of GAMEParams with one entry for each normalization type.
+   */
+  @DataProvider
+  def normalizationParameters(): Array[Array[GameParams]] =
+
+    Array(NormalizationType.values.map {
+      normalizationType =>
+        GameParams.parseFromCommandLine(CommonTestUtils.argArray(Map[String, String](
+          "task-type" -> TaskType.LINEAR_REGRESSION.toString,
+          "feature-shard-id-to-feature-section-keys-map" -> s"features:feature-0,feature-1",
+          "fixed-effect-data-configurations" -> s"global:features,1",
+          "fixed-effect-optimization-configurations" -> s"global:100,1e-11,0.3,1,LBFGS,l2",
+          "updating-sequence" -> "global",
+          "normalization-type" -> normalizationType.toString,
+          "train-input-dirs" -> "",
+          "validate-input-dirs" -> "",
+          "output-dir" -> "",
+          "feature-name-and-term-set-path" -> ""
+        )))
+    })
+
+  /**
+   * In this test, we just verify that we can fit a model with each possible normalization type, without verifying that
+   * the results are correct.
+   *
+   * Not using DataProviders, because they can't be composed to have multiple DataProviders serving arguments to
+   * a single function.
+   */
+  @Test
+  def testNormalization(): Unit = sparkTest("testNormalization") {
+
+    for (params: GameParams <- normalizationParameters()(0)) {
+
+      // This example has only a single fixed effect
+      val (coordinateId, featureShardId) = ("global", "features")
+      val labeledPoints: Seq[LabeledPoint] = trivialLabeledPoints()(0)(0)
```

Comment:
We want to use them - but here I would need two, which is not supported (and this is explained in the comment).